apis/infix-en_GB/atrium-api-infix-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/anyAssertions.kt

```diff
@@ -224,3 +227,32 @@ inline val <T> Expect<T>.it: Expect<T> get() : Expect<T> = this
  * @since 0.12.0
  */
 inline val <T> Expect<T>.its: Expect<T> get() : Expect<T> = this
+
+/**
+ * Expects that the subject of the assertion is not (equal to) [expected] and [otherValues].
+ *
+ * @return An [Expect] for the current subject of the assertion.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.13.0
+ */
+inline fun <reified T> Expect<T>.isNoneOf(expected: T, vararg otherValues: T): Expect<T> =
```

Comment:
This one we need to re-work a bit as it will not work out for the infix API.
```suggestion
infix fun <T> Expect<T>.isNoneOf(values: Values<T>): Expect<T> =
```
See other places which use`Values` and add a note how Values can be constructed to KDoc
Note that you need to add `infix` as well in order that it can be used as infix function. I guess that `inline` and `reified` are not needed here but I might be wrong.